woocommerce/woocommerce-android
# Woo Activate: Native store creation with webview checkout

This PR completes the native store creation along with a webview plan purchase. This is accomplished with the following steps:

1. Data about the store is collected (title & domain name)
2. A new site is created
3. The WP.com eCommerce monthly plan is added into a cart via API call
4. The contents of the cart is displayed in a webview
5. The user completes the purchase by selecting the payment
6. The store creation is initiated in the background (WooCommerce plugin installation & configuration)
7. After the Woo store is verified as completed, a store preview is shown

https://user-images.githubusercontent.com/1522856/203876621-2afcc398-12d0-4ca3-854d-df1feb4f81ec.mp4

**To test:**
1. Log in with a user that has $70 worth of free credits
2. Go to the site picker and tap on the `+ Add a store` button
3. Go through the steps of the flow and make a purchase
4. Verify that a working WooCommerce site was created
5. Start the store creation flow again, initiated the purchase and turn on flight mode
6. Observe the behavior and verify an appropriate error is shown
